python/pyspark/ml/feature.py

```diff
@@ -86,7 +86,7 @@ class Binarizer(JavaTransformer, HasInputCol, HasOutputCol, JavaMLReadable, Java
     """
 
     threshold = Param(Params._dummy(), "threshold",
-                      "threshold in binary classification prediction, in range [0, 1]",
+                      "Param for threshold used to binarize continuous features. The features greater than the threshold will be binarized to 1.0. The features equal to or less than the threshold will be binarized to 0.0",
```

Comment:
The style check might flag this for being too long, actually. Let's see.

Reply:
@srowen This is the same doc string as Scala's. Do you want me to shorten it?

Reply:
Just needs to be wrapped -- the line length limit is 100.

Reply:
@srowen Just updated. Could you try running the tests again please?